cmp/grid/GridModel.js

```diff
@@ -162,6 +166,10 @@ export class GridModel {
      *      `store.SummaryRecord` to be populated. Valid values are true/'top', 'bottom', or false.
      * @param {(StoreSelectionModel|Object|String)} [c.selModel] - StoreSelectionModel, or a
      *      config or string `mode` with which to create one.
+     * @param {(ColChooserModel|Object|boolean)} [c.colChooserModel] - ColChooserModel, or a
+     *      config with which to create one, or boolean `true` to enable default.
+     *      @see ColChooserModel desktop impl for valid config options.
```

Comment:
We should have a jsdoc type for what ColChooserModel can be, and do *not* need to support users actually providing an instance.   The instance is currently in the 'impl' category which suggests we are not looking to make it part of the API in its current form.